docs: toNumber built-in function (#163)
reviewpad/docs: docs/guides/built-ins/functions/toNumber/index.mdx

import CodeBlock from "@theme/CodeBlock";
import Example from "!!raw-loader!./example.yml";

**Description**:

Converts a string into a number.

**Parameters**:

| Parameter | Type   | Description                                        |
| :-------- | :----- | :------------------------------------------------- |

> | `string`  | string | The string that needs to be converted to a number. |

Reviewer: ```suggestion
| `string`  | string | The string value to convert to a number. |
```
